Headlines/src/ViewAnimator/ViewAnimator.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ViewAnimator.swift
+//  Pods-ViewAnimator_Example
+//
+//  Created by Marcos Griselli on 10/5/17.
+//
+
+import Foundation
+import UIKit
+
+// MARK: - UIView extension with animations.
+public extension UIView {
+
+    /// Performs the animation.
+    ///
+    /// - Parameters:
+    ///   - animations: Array of Animations to perform on the animation block.
+    ///   - reversed: Initial state of the animation. Reversed will start from its original position.
+    ///   - initialAlpha: Initial alpha of the view prior to the animation.
+    ///   - finalAlpha: View's alpha after the animation.
+    ///   - delay: Time Delay before the animation.
+    ///   - duration: TimeInterval the animation takes to complete.
+    ///   - dampingRatio: The damping ratio for the spring animation.
+    ///   - velocity: The initial spring velocity.
+    ///   - completion: CompletionBlock after the animation finishes.
+    func animate(animations: [Animation],
+                 reversed: Bool = false,
+                 initialAlpha: CGFloat = 0.0,
+                 finalAlpha: CGFloat = 1.0,
+                 delay: Double = 0,
+                 duration: TimeInterval = ViewAnimatorConfig.duration,
+                 usingSpringWithDamping dampingRatio: CGFloat = ViewAnimatorConfig.springDampingRatio,
+                 initialSpringVelocity velocity: CGFloat = ViewAnimatorConfig.initialSpringVelocity,
+                 options: UIView.AnimationOptions = [],
+                 completion: (() -> Void)? = nil) {
+        
+        let transformFrom = transform
+        var transformTo = transform
+        animations.forEach { transformTo = transformTo.concatenating($0.initialTransform) }
+        if !reversed {
+            transform = transformTo
+        }
+
+        alpha = initialAlpha
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewAnimator/ViewAnimator.swift#L45"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ViewAnimator.swift:45</code></td>
    </tr>
  </tbody>
</table>